.editorconfig

```diff
@@ -0,0 +1,33 @@
+# https://editorconfig.org
+# Editor Config for WebMemex
+# Download an extension for editor config to allow your IDE
+# to follow the rules of the project
+
+# Used to indicate the file is in root of the project
+root = true
+
+# Styling applicable to all forms of files
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+# Styling applicable to the file extension given
+[*.js, *.jsx]
+# This is due to the fact that github allows only 119 columns of space to view
+max_line_length = 119
```

Comment:
I intended to ask about it too; adjusting rules to Github's presentation seems somewhat arbitrary (or submissive ;) ) indeed.

I have myself tried to keep things at a soft limit of 80 characters, though I'd be fine with a larger number too. As said it is nice to be able to make longer lines if needed; I am not sure whether editors interpret such a rule as soft or hard.